source/includes/_jupyter-api-guide.html.md

```diff
@@ -47,8 +47,6 @@ Download the `.ipynb` notebook file on Bonsai's GitHub or clone the repo here: <
 
 The comments and instructions inside of the Jupyter Notebook should sufficiently explain the API calls and what to expect from the outputs but if you have any issues with this notebook please [contact support][7].
```

Comment:
if we're not fixing the API for now, we should remove this entire file, since users won't be able to use Jupyter for control or data analysis anymore.

Reply:
I undid the changes to the Jupyter page and removed links to it for now. If/when we add back API docs, we can revive this page.